linux/terraform/main.tf

```diff
@@ -106,6 +111,11 @@ module "team_servers" {
   subnet_id                   = module.vpc.public_subnets[0]
   associate_public_ip_address = true
 
+  user_data = <<-EOF
```

Comment:
Won't this only work if we run the full add user block?

```sh
useradd -m appuser || true
usermod -aG sudo appuser
printf 'appuser ALL=(ALL) NOPASSWD:ALL\n' > /etc/sudoers.d/appuser
printf 'appuser\nappuser\n' | passwd appuser
chsh --shell "$(command -v bash)" appuser #except this but makes more sense to pull out everything imo
```

Reply:
Or should it also be for the admin user? I forget the exact context.

Reply:
lawl, yeah it needs more than just that line for the team servers. I just copied the `admin` usage from the DB server when we were looking at this on Friday (`admin` on Debian already exists and has `sudo` permissions, etc).

Reply:
Oh also you know what, I DON'T want to have the appuser set up unless the init script runs, otherwise we run the risk of participants having access and not having anything set up. So this should just be `admin` as well.

Reply:
Fixed.